docker-cloud/builds/automated-build.md

```diff
@@ -105,6 +105,21 @@ run an initial test.
     Docker Cloud on every push. Only pushes to branches that are listed as the
     source for one or more tags will trigger a build.
 
+### Environment variables for builds
+
+You can set environment variables used in your build either by declaring them in
```

Comment:
My recommendation would be to keep this objective and describe how environment variables can be used to accomplish a specific use case where the user does not want to expose confidential info in their source repo (i.e. env variables can thus be declared in hooks which are accessible when those hooks are run). We're not assigning value to either approach, simply describing how a given feature can be used to accommodate a desired behavior.